### ISSUE TITLE
[Sug] For builders the same construction menu as for plants to be called

### ISSUE THREAD

**Issue by ConstantTrue** [KatekovAnton/MAXTouch-docs#53]
To regulate the speed of construction of Engeneers and Constructors

**Comment by KatekovAnton** [comment]
@ConstantTrue I cant understand this... can you rephrase? You can write in Russian if you wish

**Comment by ConstantTrue** [comment]
For LVP, HVP,  AUP, Shipyard you can see the progress of construction of a unit at any moment, but for engineer and constructor you can not.

Thus you also can change a construction speed of a plant, but you cannot change the construction speed of engeener or constructor.

**Comment by 1GodRage** [comment]
rephrase :
When producing units (with factories), each turn we can change the speed(and cost).
The actual suggestion is : want this feature for Constructor/Engineer when they are building something.

And, I guess that in order to be able to speed up the buildings, ***we may need*** the engineer/constructor to x-fer raw material during the process. Because, imo, if I have the materials at the start, then I would already build it at fast speed (x4)... And if I build it at slow speed (x1), I will probably not let extra material in the constructor/engineer. Especially if there is a risk that enemies show up before the end of the construction.


_With factories, we can also STOP the production for some turns. 😄 I guess nobody wants this feature on engineers/constructors... Thought it could give the ability to bluff the enemy: "look, I'm building for more than 6 turns, it's a missile turret !" but in fact it isn't, just a random storage unit on STOP since 5 turns... 😂 I agree, the STOP ability is not needed in MAX. 😉 _

**Comment by Hruks** [comment]
I think transfer resources to engineers/constructors should not be available because this affect game balance.
The same thing if you make custom unitset with eng/cons capacity of 200. You can build some buildings more more faster.
And this is point why units capacity is not upgradable parameter.
Of course, change speed of construction can be useful if you start on slow speed and situation on the map changed. But usual constructions units have minimum load of resources for construct building on selected speed because they not connected to base and cannot transfer resources. I think micromanagement of constructors and engineers is important part of original game strategy. It is not good idea to change this :)
